src/NHSD.BuyingCatalogue.Ordering.Domain/Order.cs

```diff
@@ -200,6 +200,16 @@ public Result Complete()
             return Result.Success();
         }
 
+        public bool DeleteOrderItem(CatalogueItemId catalogueItemId)
+        {
+            var orderItem = orderItems.SingleOrDefault(x => x.CatalogueItem.Id == catalogueItemId);
+            if (orderItem == null)
+                return false;
+
+            orderItems.Remove(orderItem);
+            return true;
```

Comment:
Can we simplify this to `return orderItems.Remove(orderItem);`?

Reply:
done